4_advanced_usage/77_distributed_rudder.txt

```diff
@@ -73,6 +73,106 @@ cd rudder-multiserver-setup
 
 ----
 
+==== Install rudder-db manually
+
+The setup is done in two places: on the database server and on the Rudder root server.
+It also allows moving an existing database to another server.
+
+===== On the database server
+
+* Install and configure the agent on the node
+
+* Install the *rudder-reports* package
+
+* Change postgresql configuration (postgresql.conf) to listen on the right interface to communicate with the server:
+
+----
+
+listen_addresses = 'IP_TO_USE' 
+----
+
+* Add an authorization line for the server (in pg_hba.conf):
+
+----
+
+host    all         all         IP_OR_NETWORK_INCLUDING_THE_SERVER  trust
+
+----
+
+* Execute the following commnd to configure the password (that should be the same than in `/opt/rudder/etc/rudder-passwords.conf` on the server) :
+
+----
+
+su - postgres -c "psql -c \"ALTER USER rudder WITH PASSWORD 'RUDDER_SERVER_DATABASE_PASSWORD'\""
+
+----
+
+* Restart postgresql to apply the new settings
+
+----
+
+service postgresql restart
+
+----
+
+===== On the root server
+
+* Remove the rudder-server-root and rudder-reports packages if installed. You can also remove the postgresql package and database from the server if existing.
```

Comment:
You should probably add a warning here that this can remove data on an existing server!